Xbim.Ifc/Extensions/IfcStoreGeometryExtensions.cs

```diff
@@ -22,9 +22,14 @@ public static class IfcStoreGeometryExtensions
         /// <param name="binaryStream">An open writable streamer.</param>
         /// <param name="products">Optional products to be written to the wexBIM file. If null, all products from the model will be saved</param>
         /// <param name="translation">Optional 3D vector to apply</param>
-        public static void SaveAsWexBim(this IModel model, BinaryWriter binaryStream, IEnumerable<IIfcProduct> products = null, 
-            IVector3D translation = null)
+        /// <param name="scale">Optional scaling factor to apply</param>
+        /// <param name="rotation">Optional rotation to apply</param>
+        public static void SaveAsWexBim(this IModel model, BinaryWriter binaryStream, IEnumerable<IIfcProduct> products = null,
+            IVector3D translation = null, double? scale = null, XbimQuaternion? rotation = null)
         {
+            if (scale == 0) throw new ArgumentException("Scale cannot be zero", nameof(scale));
+            XbimMatrix3D transformationMatrix = XbimMatrix3D.FromScaleRotationTranslation(new XbimVector3D(scale ?? 1, scale ?? 1, scale ?? 1), rotation ?? new XbimQuaternion(), translation ?? new XbimVector3D(0, 0, 0));
```

Comment:
Can we move these null coalesces to separate statements, just for clarity and so we're not repeating the check through the method. e.g. we repeat the same `scale ?? 1` on line 65

Reply:
I've moved the coalesce operations to separate lines. Thanks for the comment!